tests/DbaaS/pages/dbaasPage.js

```diff
@@ -88,6 +88,9 @@ module.exports = {
           nodesFieldErrorMessage: '$nodes-field-error-message',
           nodesNumberField: '$nodes-number-input',
           resourcesPerNode: (clusterSize) => `//label[contains(text(), "${clusterSize}")]`,
+          dbClusterExternalAccessTooltip: locate('$expose-field-container').find('.css-sr6nr'),
```

Comment:
Please don't use these css locators, they are dynamic and always change

Reply:
Ok, so what is the best way to grab this element please? I tried find it with 'svg' but it didn't work.